### DISCUSSION TITLE
URGENT!!! Need to Access the Security Onion Console FAST

### DISCUSSION THREAD

**Discussion by lconne81** [Security-Onion-Solutions/securityonion#4096] [Unsupported Versions]
Installed on VirtualBox using ISO image file version 2.3.40

I want to utilize the Security Onion Console via the IP address, but I am having trouble figuring out what IP address I need to connect to. I followed the tutorial on the Security Onion YouTube channel with some adjustments because I know it's a bit outdated. Everything else is set except for the access to the SOC. What is the surefire way to connect to the SOC?

I would appreciate a straight-to-the point answer and a fast solution because this is for a project and I don't have much time.

**Comment by nopoxan578** [comment]
Depends on your network configuration. Generally, you will need directly request the management IP after running so-allow for the IP you are connecting from.

**Reply by lconne81** [reply]
How do I request the management IP?

**Reply by nopoxan578** [reply]
Eh, in the browser :)

**Reply by lconne81** [reply]
I just look up a IP address manager to allow access???

**Reply by lconne81** [reply]
I still need to know what sort of IP address would be correct just in case. I've been using the one related to my Wi-fi connection to no avail.

**Comment by gcsrellis** [comment]
Just type in the command "ifconfig"  it will show you your IP address.  Hopefully you gave it a static IP if you are running a distributed set up.  Stand alone should be fine for DHCP.  It just may change frequently depending on your router's setup.  The interface will not be docker, lo, or veth's.
